src/App.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+import { Header } from './components/Header';
+import { InputTodo } from './components/InputTodo';
+import { TodoList } from './components/TodoList';
+import { NavBar } from './components/NavBar';
+import { v4 as uuidv4 } from 'uuid';
+
+function App() {
+  const [todoItems, setTodoItems] = useState(
```

Comment:
- [ ] OPTIONAL but recommended 👍 
    kindly always make use of the Arrow function instead of only function 👍

Reply:
sure!